### PR TITLE
Updated readme.md

### LINKED ISSUE #173: Updating the Readme of the project
**Summary:**
I will update the readme.md file of the project and make it more informative and presentable for first time contributors. The readme would also contain contribution guidelines and important links for contributing to the project.

screenshot_20190113-174944
screenshot_20190113-175035
screenshot_20190113-175100

### PR DESCRIPTION
**Pull request by codepoet2017390**
Fixes #173 
Updated the project readme so that it is easy for new contributors to set up and contribute.

To see it in action please check the readme for the project 

https://github.com/codepoet2017390/mobile-wallet/tree/Readme
(The app logo will be updated once it is available)

I have just added the basic requirements. Any comments and changes from other contributors are most welcome.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by codepoet2017390** [comment]
@ankurs287 @luckyman20  please review

File: README.md [hunk 1]
```diff
@@ -1,12 +1,79 @@
...
+
+6. You will see an error in gradle build related to missing RblClientIdProp and RblClientSecretProp. Add RblClientIdProp and RblClientSecretProp as environment variables with any value
+
+7. Sync the project again in Android studio
 
 ## Wiki
 
 https://github.com/openMF/mobile-wallet/wiki
+
+## Maintainers
```
Comment (luckyman20) [inline README.md]: Remove this part.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,79 @@
...
+<img height='175' src="https://user-images.githubusercontent.com/37406965/51083189-d5dc3a80-173b-11e9-8ca0-28015e0893ac.png" align="left" hspace="1" vspace="1">
 
-[![Join the chat at https://gitter.im/openMF/mobile-wallet](https://badges.gitter.im/openMF/mobile-wallet.svg)](https://gitter.im/openMF/mobile-wallet?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+# Mobile Wallet
 
-This repository contains a mobile wallet framework based on top of Fineract and clean architecture and contains a core library module
-that can be used as a dependency in any other wallet based project.
+Mobile wallet is an Android based frame-work for mobile wallets based on top of <a href='https://github.com/openMF/mobile-wallet/wiki/Fineract-backend'>Fineract</a>. The app follows 
+<a href='https://github.com/openMF/mobile-wallet/wiki/Architecture'>clean architecture</a> and contains a core library module
+that can be used as a dependency in any other wallet based project.It is developed at <a href='https://mifos.org/'>MIFOS</a> together with a global community.
```
Comment (luckyman20) [inline README.md]:
```suggestion
that can be used as a dependency in any other wallet based project. It is developed at <a href='https://mifos.org/'>MIFOS</a> together with a global community.
```

**Comment by ankurs287** [comment]
@codepoet2017390 Good job.

**Comment by codepoet2017390** [comment]
@ankurs287 i needed some help so how can i contact you?

**Pull request merged**